### DISCUSSION TITLE
Update Streaming Documentation Please (How to find streaming and still URLs)

### DISCUSSION THREAD

**Discussion by DabblerIOT** [tsightler/ring-mqtt#811] [General Discussion]
Thanks for sharing this project and work.  Instructions didn't show the camera still and streaming URLs as indicated here..
https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming

The procedure under overview (1, 2, 3, 4 bullets) isn't there (up to date ha running on vm image).

I was able to find them by developer tools, states, camera.[camera name].info..  saw a bunch of people recently get stuck here on different forum posts.   Got me stuck until my brain started working just a little bit the next day. :)

![Screenshot 2024-03-26 at 1 18 47 PM](https://github.com/tsightler/ring-mqtt/assets/159593555/b3f225c8-94e2-4430-85d5-b46ecad6f230)

Thanks again.

**Comment by DabblerIOT** [comment]
Fellow added a good comment to ha discussion thread..  (copy paste).. that thread hit on google search for issue I was having (not being able to find stream names in ha)

https://community.home-assistant.io/t/ring-mqtt-cant-get-video-or-snapshot-to-work/686656/3

"
for the rtsp stream, I really recommend this instead

{{ state_attr('sensor.name_of_your_ring_info', 'stream_Source') }}

note carefully… It is the sensor with the name of your ring plus ‘_info’

There is this sensor info device that is created. You can use this template for the stream URL in the generic camera config. It will pull the right URL for you.

I dont know why the videos and docs don’t do this… I find it much easier to get right
"

**Reply by tsightler** [reply]
Because this has about as much chance to be correct as to be wrong.  If it works for you, great, if it doesn't then it will never work.

**Comment by tsightler** [comment]
It's funny that you claim the info sensor entity doesn't exist while also showing a screenshot of the sensor entity existing.  If it's not showing up in the HA UI then probably you just need to force refresh your browser (long, well known issue with HA when devices are added) but that is an HA issue not a ring-mqtt issue, I can't do anything about it.

**Comment by DabblerIOT** [comment]
I understand the url is a bad guess. :)

I'm an idiot. :). Directions are close enough I shouldn't have had a problem. But...

Maybe change text a little in 2 and 3?

2. In the Sensors section of the device view select the Info sensor and, in the pop-up window, **click the Control icon in the upper right corner** - Delete this?   Before doing this I was in the right place.  When I did this, it threw me off.  (Ex: Idiot comment above.)

![Screenshot 2024-03-28 at 2 03 30 PM](https://github.com/tsightler/ring-mqtt/assets/159593555/0f146aa4-1bc9-4976-9326-75c336c2a032)

3. Select Attributes **at the bottom of the popup window** - Add this?  to expand the available attribute properties

Thanks again.

**Reply by tsightler** [reply]
I can spend my life tweaking the documentation but it doesn't stop people complaining about it, and it doesn't stop HA from making arbitrary changes to the UI every few months, so about two years ago I just stopped caring as every time HA moved a button or whatever people complained although, realistically, this project was never intended to be targeted at users who don't understand basic navigation in HA.  Rather the information was intended for people who mostly understood what was going on and could adapt.  If someone wants to maintain perfect step-by-step instructions on how to do everything with HA, I'm happy to give the maintainer rights for the wiki.

**Comment by DabblerIOT** [comment]
Cool.   Thanks for your work in sharing it.  Cheers.

**Comment by Phrana** [comment]
Hello, and I have a problem, I don't know where to get the information from the camera, I think it is because in my integration with HA the camera does not appear.
![ring](https://github.com/tsightler/ring-mqtt/assets/118558930/976ed0a0-dc1c-4a7b-a5b1-681e6677665b)
